sdk/src/main/java/io/opentelemetry/sdk/metrics/data/MetricData.java

```diff
@@ -275,6 +273,24 @@ public static ValueAtPercentile create(double percentile, double value) {
       SUMMARY,
     }
 
+    /**
+     * A description of the temporal nature of the aggregation that was performed to produce the
+     * MetricData.
+     */
+    public enum TemporalQuality {
+      /**
+       * The MetricData is relevant for a single moment in time. This is most common for metrics
+       * generated from Observer Instruments
+       */
+      INSTANTANEOUS,
+
+      /** The MetricData is aggregated over the lifetime of the instrument that created it. */
+      CUMULATIVE,
+
+      /** The MetricData is aggregated since the last collection cycle. */
+      DELTA
+    }
```

Comment:
I like how stackdriver defines this https://cloud.google.com/monitoring/api/v3/kinds-and-types, this is a kind and a PointType (valueType).

Also if we go with that model non-monotonic values can be only INSTANTANEOUS and monotonic values can be only CUMULATIVE or DELTA. What do you think?

Reply:
In other words I think we don't need to have monotonic/non-monotonic in the possible Types, only LONG_VALUE, DOUBLE_VALUE is enough.

Reply:
What you suggest fits New Relic's data model, for sure. I don't know enough about other backends to know if it's complete enough for all cases. Seems like a good topic for next week's metrics SIG (which I'll miss since I'm on vacation).